### PR TITLE
Issue #21635: Update Servlet 6.0 factories bundle name.

### LINKED ISSUE #21635: Rename `io.openliberty.servlet.6.0.internal.factories` to `io.openliberty.webcontainer.servlet.6.0.internal.factories`
Rename for consistency.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #21635

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Review by pmd1nh — APPROVED** [review]

**Review by volosied — COMMENTED** [review]
The package info file was deleted because that particular package is not exported. Is my understanding correct? 
https://github.com/OpenLiberty/open-liberty/blob/b1797ecc908e847c29a3d18732fb8c993a74eb81/dev/io.openliberty.servlet.6.0.internal.factories/bnd.bnd#L64

File: dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/bnd.bnd [hunk 1]
```diff
@@ -52,17 +52,17 @@ Service-Component: \
...
-    !io.openliberty.servlet.internal.session.impl.factory, \
-    !io.openliberty.servlet.internal.util.factory, \
+    !io.openliberty.webcontainer.servlet.internal.async.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.container.config.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.response.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.srt.factory, \
+    !io.openliberty.webcontainer.servlet.internal.osgi.webapp.factory, \
+    !io.openliberty.webcontainer.servlet.internal.servlet.factory, \
+    !io.openliberty.webcontainer.servlet.internal.session.impl.factory, \
+    !io.openliberty.webcontainer.servlet.internal.util.factory, \
```
Comment (volosied) [inline dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/bnd.bnd:65]: Can't comment any lower, but could you fix the indent for `com.ibm.ws.kernel.boot.core;version=latest,\` below?  We can get it later though.

**Review by volosied — APPROVED** [review]

File: dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/bnd.bnd (hunk 1 above)
Comment (pnicolucci) [reply dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/bnd.bnd:65]: I've pushed up some minor formatting corrections.

**Comment by pnicolucci** [comment]
> The package info file was deleted because that particular package is not exported. Is my understanding correct?
> 
> https://github.com/OpenLiberty/open-liberty/blob/b1797ecc908e847c29a3d18732fb8c993a74eb81/dev/io.openliberty.servlet.6.0.internal.factories/bnd.bnd#L64

@volosied 

Each package still has a `package-info.java` this is just something odd git is doing in the PR comparison. I tried to do this PR twi times and it did the same thing each time.

Summary:

**deleted:**
`dev/io.openliberty.servlet.6.0.internal.factories/src/io/openliberty/servlet/internal/session/impl/factory/package-info.java`


**renamed:**
`dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/src/io/openliberty/webcontainer/servlet/internal/osgi/response/factory/package-info.java`
**to**
`dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/src/io/openliberty/webcontainer/servlet/internal/session/impl/factory/package-info.java`

**new:**
`dev/io.openliberty.webcontainer.servlet.6.0.internal.factories/src/io/openliberty/webcontainer/servlet/internal/osgi/response/factory/package-info.java`

**Comment by pnicolucci** [comment]
#libby #build

**Pull request merged**